backends/advanced/init.py

```
@@ -347,6 +380,12 @@ def setup_optional_services(self):
             self.config["PARAKEET_ASR_URL"] = self.args.parakeet_asr_url
             self.console.print(f"[green][SUCCESS][/green] Parakeet ASR configured via args: {self.args.parakeet_asr_url}")
 
+        # Check if Tailscale auth key provided via args
+        if hasattr(self.args, 'ts_authkey') and self.args.ts_authkey:
+            self.config["TS_AUTHKEY"] = self.args.ts_authkey
+            self.console.print(f"[green][SUCCESS][/green] Tailscale auth key configured (Docker integration enabled)")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)